spec/views/spree/admin/payment_methods/index.html.haml_spec.rb

```diff
@@ -19,10 +21,52 @@ def object_url(object = nil, options = {}) "" end
   end
 
   describe "payment methods index page" do
-    it "shows only the providers of the existing payment methods" do
-      render
+    context "when user is not admin" do
+      before do
+        allow(view).to receive_messages spree_current_user: create_enterprise_user
+      end
+
+      it "shows only the providers of the existing payment methods" do
+        render
+
+        expect(rendered).to have_content "Cash/EFT/etc. (payments for which automatic validation is not required)", count: 2
+      end
+
+      it "does not show Enviroment column when user is not admin" do
```

Comment:
you can avoid `when user is not admin` because that's exactly what this test's context says

Reply:
Done